instance/newrelic.py

```diff
@@ -84,11 +85,53 @@ def create_synthetics_monitor(uri, name=None, monitor_type='SIMPLE',
         'frequency': frequency,
         'locations': locations,
         'status': 'ENABLED',
+        'options': {'verifySSL': True},
+    })
+    r.raise_for_status()
+    return r.headers['location'].rsplit('/', 1)[-1]
+
+
+def create_synthetics_ssl_monitor(name=None, monitor_type='SCRIPT_API',
+                                  frequency=1440, locations=('AWS_US_EAST_1',)):
+    """
+    Create a SSL monitor.
+    """
+    url = '{0}/monitors'.format(SYNTHETICS_API_URL)
+    logger.info('POST %s', url)
+    r = requests.post(url, headers=_request_headers(), json={
+        'name': name,
+        'type': monitor_type,
+        'frequency': frequency,
+        'locations': locations,
+        'status': 'ENABLED',
     })
     r.raise_for_status()
     return r.headers['location'].rsplit('/', 1)[-1]
 
 
+def update_synthetics_ssl_monitor(monitor_id, url_to_check):
+    """
+    Update the ssl monitor script
+    """
+    url = '{0}/monitors/{1}/script'.format(SYNTHETICS_API_URL, monitor_id)
+    logger.info('PUT %s', url)
+    headers = _request_headers()
+    headers['Content-Type'] = 'application/json'
+    current_directory = os.path.dirname(os.path.realpath(__file__))
+    script_file = os.path.join(current_directory, 'scripts', 'check_ssl_expiration.js')
+    with open(script_file, 'r') as f:
```

Comment:
Maybe it could be cleaner to use the Django template engine? Could be overkill too, feel free to ignore my comment. :)